src/objc/messages.rs

```diff
@@ -249,6 +399,53 @@ where
 /// See also [msg_class], if you want to send a message to a class.
 #[macro_export]
 macro_rules! msg {
+    [$env:expr; $receiver:tt $name:ident $(: $arg1:tt)?
+                             $($namen:ident: $argn:tt)* $(,$va_argn:ident)*] => {
+        {
+            let sel = $crate::objc::selector!($($arg1;)? $name $(, $namen)*);
+            let sel = $env.objc.lookup_selector(sel)
+                .expect("Unknown selector");
+            if cfg!(debug_assertions)
+                && $env.cpu.regs_mut()[$crate::cpu::Cpu::SP]
+                    > $crate::mem::guest_size_of::<$crate::objc::objc_super_debug>()
```

Comment:
If the stack pointer has gotten low enough that this check would make sense, it's already overwritten the entire app binary and more. I guess this functions as an overflow check, but since you're only doing this in debug builds, the default Rust overflow checks will apply anyway. So I would suggest just removing this.

Reply:
Fair enough. I had this for the late linking part (up above), and thought the error message was a bit unintuitive. I can change it back though.

Reply:
With that said, I don't think it makes sense for `msg_super!` since that won't work in linking either way, so I guess that can go.

Reply:
Got rid of it entirely, I don't think it'll be a common enough issue to matter.